src/Components/Scenes/RequestAssets/RequestingFor/RequestingFor.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Select, { createFilter } from 'react-select';
+import { makeStyles } from '@material-ui/core/styles';
+import MenuList from './MenuList';
+import { Typography } from '@material-ui/core';
+
+const useStyles = makeStyles({
+  table: {
+    width: 400
+  },
+  menuStyle: {
+    width: 'max-content',
+    textTransform: 'uppercase',
+    marginBottom: 8
+  }
+});
+
+const customStyles = {
+  option: (provided, state) => ({
+    ...provided,
+    fontSize: 12
```

Comment:
@sfowles font sizes, and colors should all be using Design tokens, no?

Reply:
Not fontSize but color yes.

Reply:
added fontsize